### PR TITLE
added "modal" to sidenav in every page.

### PR DESCRIPTION
**Pull request by zoyatanweer**
- added "modal" component in side navigation of every page
- fixed the badge component.
- added installation link and footer to the landing page

### CODE REVIEW

**Review by rutu14 — APPROVED** [review]

File: components/avatar/avatar.html [hunk 1]
```diff
@@ -13,32 +13,32 @@
 
 <body>
     <nav class="navigation-container">
-        <div class="nav-brand">FALCON</div>
+        <div class="nav-brand">FALCON UI</div>
         <ul class="list-non-bullet nav-pills">
             <li class="list-item">
-                <a class="link link-active" href="">HOME</a>
+                <a class="link link-active" href="/index.html">HOME</a>
```
Comment (dev-prabhat) [inline components/avatar/avatar.html:19]: you can also add href="/" i think it works fine

**Review by dev-prabhat — CHANGES_REQUESTED** [review]
Looks good to me

**Comment by bossmonkey98** [comment]
Try to make it responsive

**Review by bossmonkey98 — APPROVED** [review]

File: components/badge/badge.css [hunk 2]
```diff
@@ -1,58 +1,61 @@
...
-    color: white;
-}
-.btn-icon-home{
-    left:345px;
-}
-.btn-icon-email{
-    left:400px;
+    position: absolute;
+    top: -5px;
+    left: 25px;
```
Comment (zoef-yt) [inline components/badge/badge.css]: use rem.

**Review by zoef-yt — APPROVED** [review]
The code looks good. Good to merge with the minor change.

**Review by Muzammil-khan710 — APPROVED** [review]

**Comment by zoyatanweer** [comment]
- added installation link in index.html
- changed the secondary color of button 
- changed the hover state of outlined button

**Pull request merged**